insights/Extract_MC_Insights_DBX.py

```diff
@@ -9,19 +9,49 @@
 
 # COMMAND ----------
 
+# DBTITLE 1,Create User Input Widgets
+insight_name_to_report_mapping = {
+    "key_assets": "key_assets.csv",
+    "monitors": "monitors.csv",
+    "cleanup_suggestions": "cleanup_suggestions.csv",
+    "field_cleanup_suggestions": "field_level_cleanup_suggestions.csv",
+    "field_importance_scores": "field_importance_scores.csv",
+    "events": "events.csv",
+    "table_read_write_stats": "table_read_write_stats.csv",
+    "query_runtime_trends": "query_runtime_trends.csv",
+    "rule_and_sli_results": "rule_and_sli_results.csv",
+    "insight_monitor_recom_dt_fields": "insight_monitor_recom_dt_fields.csv",
+    "insight_monitor_recom_fh_tables": "insight_monitor_recom_fh_tables.csv",
+    "insight_monitor_issues_and_solutions": "insight_monitor_issues_and_solutions.csv",
+    "insight_inactive_table_monitors": "insight_inactive_table_monitors.csv",
+    "bi_dashboard_analytics": "bi_dashboard_analytics.csv",
+    "incident_history": "incident_history.csv",
+    "heavy_queries": "heavy_queries.csv",
+    "consumption_by_user": "consumption_by_user.csv",
+    "custom_monitor_notifications": "custom_monitor_notifications.csv",
+    "unmonitored_tables": "unmonitored_tables.csv",
+}
```

Comment:
Let's pull this from the API - https://apidocs.getmontecarlo.com/#query-getInsights

That way this mapping doesn't need to be hard coded / maintained.

Reply:
good idea, implemented